test/src-symlinks.js

```diff
@@ -55,13 +59,19 @@ describe('.src() with symlinks', function() {
       expect(files[0].stat.isFile()).toEqual(true);
     }
 
-    pipe([
+    pipeline([
       vfs.src(symlinkNestedFirst),
       concat(assert),
     ], done);
   });
 
   it('resolves directory symlinks correctly', function(done) {
+    // Error: EPERM: operation not permitted
+    if (os.platform() === 'win32' && nodeVersion.major === 10) {
+      this.skip();
+      return;
+    }
```

Comment:
I want to look into this to understand it

Reply:
This seems to be a bug with node 10 but we want to support Node 10 on Windows so I've written a workaround. I left a TODO to remove the recursive readlink when we drop Node 10 and I'll open a follow up issue.